ui/src/components/Profile.scss

```
@@ -0,0 +1,25 @@
+.profile {
```

".profile" is quite generic name and it is highly likely to collide. As a plugin, we should avoid using generic classnames.

```suggestion
.q-profile {
```